src/main/java/net/dv8tion/jda/api/requests/restaction/MessageAction.java

```diff
@@ -844,10 +846,13 @@ default MessageAction retainFiles(@Nonnull Collection<? extends Message.Attachme
      */
     @Nonnull
     @CheckReturnValue
-    default MessageAction setActionRows(@Nonnull Collection<? extends ActionRow> rows)
+    default MessageAction setActionRows(@Nonnull Collection<? extends ComponentLayout> components)
```

Comment:
This is a breaking change nobody asked for, revert it.

Reply:
What's the point in accepting ComponentLayouts if MessageAction only accepts ActionRows?

Reply:
A robust interface.

Reply:
so `editComponentsById()` in MessageChannel should also accept ActionRows?

Reply:
No. Please just do what i asked?

Reply:
Yeah but `editComponentyById` accepts ComponentLayout although MessageActionImpl only accepts ActionRows. Should i cast it? Maybe i'm lost but you're confusing me lol